pages/Admins/index.tsx

```diff
@@ -1,8 +1,26 @@
+import * as React from 'react';
```

Comment:
LGTM!! Just please fix this import. In general, I discourage the use of wildcard imports, just import what you need.
```suggestion
import React from 'react';
```